### ISSUE TITLE
[2.1]: Multiple instances of sceditor break

### ISSUE THREAD

**Issue by asmith20002** [SimpleMachines/SMF#7970]
### Basic Information

The sceditor is setup like this in Source of the pages:

```
create_control_richedit($editorOptions);
```

and in template pages:
```
template_control_richedit($context['post_box_name'], 'smileyBox_message', 'bbcBox_message');
```

However, if a mod is trying to add 2 separate sceditors on the same page they would simply duplicate the code above which almost works. It used to work in 2.0 perfectly fine but in sceditor it breaks due to how it is being setup in `jquery.sceditor.smf.js`.

There are a few `document.querySelector(".sceditor-container")` or `$('.sceditor-container')` in there which are preventing sceditors isolation.

### Steps to reproduce

1.
2.

### Version/Git revision

2.1

### Database Engine

All

**Comment by dragomano** [comment]
What kind of modification are you talking about?

**Comment by asmith20002** [comment]
I made a custom page on SMF with one simple source file and one template file. I wanted to have 2 textareas with sceditor:

source sample:
```
create_control_richedit($editorOptions);
create_control_richedit($editorOptions2);
```

and also ran the template function:

```
template_control_richedit($first_id,  'smileyBox_message', 'bbcBox_message');
template_control_richedit($second_id,  'smileyBox_message', 'bbcBox_message');
```

The result is 2 buggy sceditors.  For example the smiley sets are shown thrice. (instead of twice) and when you click on a smiley on the top editor, it gets inserted into the bottom editor.

Anyway I have fixed the problem. There's a PR.
